### PR TITLE
Include devtools in wrangler monorepo

### LINKED ISSUE #881: Formalize devtools
We should formalize [built-devtools](https://github.com/threepointone/built-devtools/) as a part of the wrangler repo

- [x] #1416
- [ ] Automatically deploy the built output somewhere
- [ ] Related: make that "somewhere" something official looking (e.g. not pages.dev)
- [x] #607

### LINKED ISSUE #1416: Include a package that renders chrome-devtools-frontend

### PR DESCRIPTION
**Pull request by caass**
Previously, wrangler relied on @threepointone's [built-devtools](https://github.com/threepointone/built-devtools). This PR includes these devtools in the wrangler repository.

`packages/wrangler-devtools` contains a static-asset worker that serves the output of building `chrome-devtools`.

`npm run -w wrangler-devtools publish` will build & publish `wrangler-devtools.<my-subdomain>.workers.dev`.

Closes #1416

### CODE REVIEW

File: packages/wrangler-devtools/wrangler.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name = "wrangler-devtools"
```
Comment (GregBrimble) [inline packages/wrangler-devtools/wrangler.toml:1]: Pages doesn't listen to `wrangler.toml`. You should add `--project-name=wrangler-devtools` to your `publish` command if this is the project name you are using.

**Comment by caass** [comment]
This continues to not work because `--assets` isn't happy about the size of some of the files.

File: packages/wrangler-devtools/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+built-devtools
```
Comment (threepointone) [inline packages/wrangler-devtools/.gitignore:1]:
```suggestion
built-devtools

```

**Review by threepointone — COMMENTED** [review]
It looks good! Some questions, I'll approve after they're clarified?

File: packages/wrangler-devtools/wrangler.toml [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+compatibility_date = "2022-07-01"
```
[the remark this answers is not in the document]
Comment (caass) [reply packages/wrangler-devtools/wrangler.toml:2]: I think it'd be better to do this as an environment variable in a CI step

File: packages/wrangler-devtools/package.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	"version": "0.0.0",
+	"author": "wrangler@cloudflare.com",
+	"description": "Chrome Devtools hosted for easy use with Wrangler",
+	"license": "BSD-3-Clause",
+	"bugs": {
+		"url": "https://github.com/cloudflare/wrangler2/issues"
+	},
+	"homepage": "https://github.com/cloudflare/wrangler2#readme",
+	"scripts": {
+		"publish": "wrangler publish --assets ./built-devtools",
```
Comment (caass) [inline packages/wrangler-devtools/package.json:13]:
```suggestion
		"publish": "npx wrangler publish --assets ./built-devtools",
```

**Review by threepointone — COMMENTED** [review]
I can't see anything here anymore lol. But it looks like there's no package.json under packages/wranger-devtools? Or is that just github being weird? If so, let's land this.

**Comment by caass** [comment]
It's probably easier to review by checking it out on your machine -- the package.json is still there 👍🏻

**Comment by threepointone** [comment]
Can't stamp this on my phone, but let me merge it in anyway

**Pull request merged**